doc/source/cluster/kubernetes/user-guides.md

```diff
@@ -14,6 +14,8 @@ user-guides/storage
 user-guides/config
 user-guides/configuring-autoscaling
 user-guides/kuberay-gcs-ft
+user-guides/kuberay-gcs-ft
```

Comment:
Can we add a note in the original guide that references the new guide for persistence options?

Reply:
Done.